docs/dev/cloud-config-sync.md

```diff
@@ -0,0 +1,36 @@
+# Managing cloud-config for CCMs
+
+# Intro
+
+Some cloud providers,
+such as [Azure](https://kubernetes-sigs.github.io/cloud-provider-azure/install/configs/) or [vSphere](https://cloud-provider-vsphere.sigs.k8s.io/cloud_config.html),
+require config file which contains various platform-specific parameters (e.g. api endpoints, resource group name or so).
+
+In OpenShift terms this config represented as ConfigMap defined during the installation procedure and managing by [cluster-config-operator](https://github.com/openshift/cluster-config-operator).
+
+There are two places where this config map stored on a running cluster at the moment (OCP 4.8):
+1. `kube-cloud-config` ConfigMap in `openshift-config-managed` namespace
+2. ConfigMap with arbitrary name in `openshift-config` namespace. Such name might be taken from `cluster` Infrastructure resource spec.
```

Comment:
is this an automatically generated name?

Reply:
Don't know to be honest. 
https://docs.openshift.com/container-platform/4.7/rest_api/config_apis/infrastructure-config-openshift-io-v1.html#specification
Didn't find from where it comes. Judging by Infrastructure resource schema - it might be arbitrary string.

Reply:
It is a field in the spec of the user managed resource. It is definitely something that could be different from installation to installation.